src/components/BattleScreen/BattleDisplay/BattleDisplay.js

```diff
@@ -0,0 +1,54 @@
+import { useSelector } from "react-redux";
+import classes from "./BattleDisplay.module.scss";
+
+const BattleDisplay = (props) => {
+  const user = useSelector((state) => state.user.user);
+  const enemy = useSelector((state) => state.enemy.enemy);
+
+  return (
+    <div className={classes.display}>
+      <div className={classes.enemy_hp}>
+        <p>{enemy.firstEnemy.forms[0].name}</p>
+        <progress
+          className={classes.hp_bar}
+          id="hp"
+          value="100"
+          max="100"
+        ></progress>
+      </div>
+      <div className={classes.user_image}>
+        <img
```

Comment:
Make sure you add alt values for each img.

Reply:
Thanks, dealt with it